waspc/src/Wasp/Generator/Crud/Routes.hs

```diff
@@ -0,0 +1,18 @@
+module Wasp.Generator.Crud.Routes where
+
+import Data.List (intercalate)
+import qualified Wasp.AppSpec.Crud as AS.Crud
+
+getRoute :: AS.Crud.CrudOperation -> String
+getRoute operation = case operation of
+  AS.Crud.Get -> "get"
+  AS.Crud.GetAll -> "get-all"
+  AS.Crud.Create -> "create"
+  AS.Crud.Update -> "update"
+  AS.Crud.Delete -> "delete"
+
+makeFullPath :: String -> AS.Crud.CrudOperation -> String
+makeFullPath crudOperationName crudOperation = intercalate "/" [getCrudOperationRouterRoute crudOperationName, getRoute crudOperation]
```

Comment:
It's a Express route :) I think the implementation is not complicated and I wouldn't want to go into "construct a posix path" then "convert the path to string" route, I think it would look more complicated than it needs to be.